publication/ver11/cr/security_and_privacy.md

```diff
@@ -0,0 +1,785 @@
+under Thing Description.
+
+When appropriate we will cross-link our responses to responses associated with these other documents.
+
+Please raise issues specific to specific documents on the appropriate issue tracker:
+* [Web of Things (WoT) Architecture GitHub Issue page](https://github.com/w3c/wot-architecture/issues)
+* [Web of Things (WoT) Thing Description GitHub Issue page](https://github.com/w3c/wot-thing-description/issues)
+* [Web of Things (WoT) Discovery GitHub Issue page](https://github.com/w3c/wot-discovery/issues)
+* [Web of Things (WoT) Profile GitHub Issue page](https://github.com/w3c/wot-profile/issues)
+
+## Background
+
+The [WoT Security and Privacy Considerations](https://github.com/w3c/wot-security) document
+includes an extensive threat model,
+and considers both passive and active attacks to
+both data (payload transactions) and metadata (Thing Descriptions).
+It should be made clear however that the WoT is _descriptive_.
+WoT Thing Descriptions can be used to describe existing IoT devices.
+In this case the WoT
+is subject the same threats and mitigations as those devices.
+For "new" devices written from scratch to conform to the WoT specifications,
+we define a set of 
+[WoT Security Best Practices](https://github.com/w3c/wot-security-best-practices)
+which will be updated as necessary (for example, as the WoT is extended to new
+IoT protocols or security schemes via its built-in extension mechanisms).
+In addition, we define a general security testing procedure for WoT implementations in the
+[WoT Security Testing Plan](https://github.com/w3c/wot-security-testing-plan).
+
+In the documents above we address the threats and attack surfaces of the
+WoT in detail.  In the following we have answered the questions posed by the
+W3C Security and Privacy Questionnaire as best as we could... but please take into consideration
+that most of these questions are not fully applicable to our situation.
+
+Security in the IoT is unavoidably more complex than in the web.
+IoT systems often use patterns of communication that are more general
+than the relatively straightforward client-server architecture of web servers and browsers.
+For example, the "same origin policy" is considered a cornerstone of web security.
+However, a single IoT device may connect and integrate information from several other
+IoT devices (and even send commands to them) as a fundamental part of its operation.
+In addition,
+security on the web is often concerned with the "user's device" or the "user agent",
+which is always a single client (the browser).
+The IoT is often more concerned with machine-to-machine communication;
+there is frequently not even a human "user" directly associated with a device.
+If an IoT device _is_ associated with a user, such as in a Smart Home context,
+the device may be a client, or
+a server---or even both at the same time.
+Devices may also be associated with many users, not just one.
+Finally, even if an IoT device has
```

Comment:
- IoT devices can have their own user interfaces, even if these are just blinking lights and buttons.
- IoT hubs can provide user interfaces, e.g. dashboards, that are often rendered using HTML (which is why the XSS script injection risk is relevant for strings in TDs)

I general, though, the questions need some work to understand what they "mean" in our context, since most are written under the assumption our spec is defining a browser API or extension.  Unfortunately.

I could certainly add some more text to the introduction trying to position ourselves regarding this problem in particular, although I don't want to make the intro TOO verbose.

Reply:
added a bit more text.  My point was actually that we CAN'T in general address issues related to the UI.